config.go

```diff
@@ -274,6 +275,16 @@ func convert(data string, value reflect.Value) error {
 		value.SetInt(int64(d))
 		return nil
 	}
+
+	if t == timeType {
+		d, err := time.Parse(time.RFC3339, data)
```

Comment:
Probably worth documenting this expected format in the README, for example [here](https://github.com/heetch/confita/blob/88e836f3254fac9e7e3dfc55eea371769acc708b/README.md?plain=1#L34-L42).